src/pages/fuel.tsx

```diff
@@ -8,10 +9,13 @@ export default function FuelConsumption() {
         <title>Fuel Consumption</title>
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
-      <nav className="flex justify-center gap-4">
-        <Link href='/'>Home</Link>
-      </nav>
-      <h2>Fuel Consumption Page</h2>
+      <picture>
```

Comment:
Not really a priority, but I suggest you use NextJS's `Image` component for pictures. That way, you can have your image link as an import and that image link then associated with a variable.  You can then pass that variable name into the `src` prop of `Image`.

`<Image src={<variable name here>} />

Reply:
I tried using Image, but I got an error.